### PR TITLE
add label and note color change

### PR DESCRIPTION
**Pull request by MurtazaBagwala07**
User can add label to a note
A label page which shows notes with diff labels
User can change background color of NoteCard

### CODE REVIEW

File: memento-notes/src/components/InputNote/InputNote.jsx [hunk 1]
```diff
@@ -11,20 +13,26 @@ export const InputNote = () => {
...
     })
 
     const changeHandler = (e) => {
         setInputNote({ ...inputNote, [e.target.name]: e.target.value });
+        console.log(inputNote);
       };
 
       const submitHandler=async(e)=>{
         e.preventDefault();
+        let random =colors[Math.floor(Math.random()*colors.length)]
```
Comment (OmKakatkar) [inline memento-notes/src/components/InputNote/InputNote.jsx]: Is the `let` here required?

File: memento-notes/src/components/InputNote/InputNote.jsx [hunk 2]
```diff
@@ -11,20 +13,26 @@ export const InputNote = () => {
...
+        setInputNote({ ...inputNote,time:`${new Date(Date.now()).toLocaleDateString()}`, color:random})
         const response = await AddNoteService(inputNote,auth.token);
         dispatch({
           type:"SET_NOTES",
           payload:response
         })
+        console.log(response)
```
Comment (OmKakatkar) [inline memento-notes/src/components/InputNote/InputNote.jsx:34]: Remove console if not required

**Review by OmKakatkar — APPROVED** [review]
Like:
* The use of color variables
* Code is modular

Suggestion;
* Add a hosted link for PR. Makes it easy to review and check the functionality as well.

File: memento-notes/src/components/InputNote/InputNote.jsx (hunk 1 above)
Comment (MurtazaBagwala07) [reply memento-notes/src/components/InputNote/InputNote.jsx]: will change it to const

**Review by rahulbakshi4 — APPROVED** [review]
- Code seems good
- You can give variable names to color rather than direct hex value
- Add a preview a screenshot or hosted link for better review

**Pull request merged**